servers/xr/xr_interface.cpp

```diff
@@ -72,6 +72,8 @@ void XRInterface::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_passthrough_enabled"), &XRInterface::is_passthrough_enabled);
 	ClassDB::bind_method(D_METHOD("start_passthrough"), &XRInterface::start_passthrough);
 	ClassDB::bind_method(D_METHOD("stop_passthrough"), &XRInterface::stop_passthrough);
+	ClassDB::bind_method(D_METHOD("get_transform_for_view", "view", "cam_transform"), &XRInterface::get_transform_for_view);
+	ClassDB::bind_method(D_METHOD("get_projection_for_view", "view", "aspect", "near", "far"), &XRInterface::get_projection_for_view);
```

Comment:
"near" and "far" should be "z_near" and "z_far". This'll need to be updated in the API docs as well.

Reply:
No, "near" and "far" are the preferred names, they were renamed awhile ago in https://github.com/godotengine/godot/pull/44434

Reply:
Ah, ok, thanks! Sorry for adding confusion here :-/